UniConnectApp/app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.uniconnect">
 
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
Later when you add an interface, the app will want this permission to hit the API